TombEngine/Objects/Generic/Object/Pushable/PushableCollision.cpp

```diff
@@ -28,24 +30,24 @@ namespace TEN::Entities::Generic
 		if (pushableItem.Status == ITEM_INVISIBLE || pushableItem.TriggerFlags < 0)
 			return false;
 
-		auto pointColl = CollisionResult{};
+		auto pointColl = GetPointCollision(pushableItem);
```

Comment:
Why you are doing `GetPointCollision()` here, and immediately down the line, inside else block?